### ISSUE TITLE
"Element type is invalid: expected a string..." Error when used with Gatsby

### ISSUE THREAD

**Issue by DigitalGoldfish** [jaredpalmer/formik#785]
## Current Behavior

Page does not render; error in console:

Uncaught Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.
    at invariant (VM11577 commons.js:17384)
    at ReactCompositeComponentWrapper.instantiateReactComponent [as _instantiateReactComponent] (VM11577 commons.js:56591)
    at ReactCompositeComponentWrapper.performInitialMount (VM11577 commons.js:46079)
    at ReactCompositeComponentWrapper.mountComponent (VM11577 commons.js:45970)
    at Object.mountComponent (VM11577 commons.js:52402)

![image](https://user-images.githubusercontent.com/559330/43137223-8d4d1334-8f4b-11e8-85a4-2c316754eb3f.png)

## Steps to Reproduce
 - yarn add formik
 - Copy one of the example forms linked in the README  into the gatsby app as a component
 - gatsby develop
 - Navigate to the page

I will try to verify later if it happens in a new gatsby install or only for this page

## Expected behavior

Form should be rendered

## Suggested solution(s)

Downgrading formik to 0.11.11 (or one of the 1.0.0.alpha versions solves the problem and the form displays as expected)

## Additional context

Gatsby 1.9.277
Gatsby-plugin-react-next 1.0.11 

**CodeSandbox Link**:

---

- Formik Version: every version newer than 1.0.0.alpha6
- React Version: 16.4.1
- TypeScript Version: -
- Browser and Version: Chrome 67.0.3396.99
- OS: Windows 10
- Node Version: 10.1.0
- Package Manager and Version: yarn v1.7.0

**Comment by DigitalGoldfish** [comment]
I tried to reproduce with a brand new gatsby installation and there it was working right away. Exactly the same form code that I used for my earlier test. Going back to my initial installation I upgrade formik again to the latest version (1.0.1) and suddenly it started working. 
I am not sure why though ... since the code is still unchanged it must either have something to do with cleaning the gatsby cache or that i used yarn upgrade to check for newer version of dependencies.

In any case - nothing wrong with formik. I will close.
